### PR TITLE
add switch for implicit datatype conversion

### PR DESCRIPTION
**Pull request by Nicole00**

### CODE REVIEW

File: tools/exchange/src/main/resources/application.conf [hunk 1]
```diff
@@ -58,6 +58,22 @@
...
       partition: 32
+      # isImplicit: As a switch to allow implicit conversion of data, default true
+      # if isImplicit is true, the source field's datatype will be convert according to nebula data type.
+      #
+      # for Nebula String type
+      #     1. empty String will be convert to ""
+      #     2. Short, Int, Long, Decimal, Float, Double will be convert to String
+      #     3. Boolean true will be convert to String as format "true"
+      #     4. List/Array will be convert to String as format: [value1,value2]
+      #     5. Map will be convert to String as format: {key1:value1,key2:value2}
```
Comment (laura-ding) [inline tools/exchange/src/main/resources/application.conf:69]: may be is `{"key1":value1,"key2":value2}`

File: tools/exchange/src/main/scala/com/vesoft/nebula/tools/importer/processor/Processor.scala [hunk 2]
```diff
@@ -58,185 +58,202 @@ trait Processor extends Serializable {
...
+      row.schema.fields(index).dataType match {
+        case StringType => {
+          val result = row.getString(index).mkString("\"", "", "\"")
+          if (toBytes) result.getBytes else result
+        }
+        case _: Any => {
+          row.get(index)
         }
       }
+    } else {
```
Comment (laura-ding) [inline tools/exchange/src/main/scala/com/vesoft/nebula/tools/importer/processor/Processor.scala:80]: You can add `return` in the if, so it no need else

File: tools/exchange/src/main/resources/application.conf (hunk 1 above)
Comment (Nicole00) [reply tools/exchange/src/main/resources/application.conf:69]: it's "{key1:value1,key2:value2}"

File: tools/exchange/src/main/scala/com/vesoft/nebula/tools/importer/config/SchemaConfigs.scala [hunk 3]
```diff
@@ -63,7 +66,8 @@ case class TagConfigEntry(override val name: String,
       s"vertex field ${vertexField} " +
       s"vertex policy ${vertexPolicy} " +
       s"batch ${batch} " +
-      s"partition ${partition}"
+      s"partition ${partition}" +
```
Comment (darionyaphet) [inline tools/exchange/src/main/scala/com/vesoft/nebula/tools/importer/config/SchemaConfigs.scala:69]: append an extra space

File: tools/exchange/src/main/scala/com/vesoft/nebula/tools/importer/config/SchemaConfigs.scala [hunk 4]
```diff
@@ -131,7 +137,8 @@ case class EdgeConfigEntry(override val name: String,
         s"target field ${targetField} " +
         s"target policy ${targetPolicy} " +
         s"batch ${batch} " +
-        s"partition ${partition}"
+        s"partition ${partition}" +
```
Comment (darionyaphet) [inline tools/exchange/src/main/scala/com/vesoft/nebula/tools/importer/config/SchemaConfigs.scala:140]: ditto

File: tools/exchange/src/main/scala/com/vesoft/nebula/tools/importer/config/SchemaConfigs.scala [hunk 5]
```diff
@@ -120,7 +125,8 @@ case class EdgeConfigEntry(override val name: String,
...
```
Comment (darionyaphet) [inline tools/exchange/src/main/scala/com/vesoft/nebula/tools/importer/config/SchemaConfigs.scala:128]: ditto

**Comment by darionyaphet** [comment]
Generally LGTM

**Review by darionyaphet — APPROVED** [review]

**Pull request merged**